cmd/fyne_demo/data/metadata_bundled.go

```diff
@@ -0,0 +1,12 @@
+// auto-generated
+// Code generated by '$ fyne bundle'. DO NOT EDIT.
+
+package data
+
+import "fyne.io/fyne/v2"
+
+var resourceAuthors = &fyne.StaticResource{
+	StaticName: "AUTHORS",
+	StaticContent: []byte(
+		"Andy Williams <andy@andy.xyz>\nSteve OConnor <steveoc64@gmail.com>\nLuca Corbo <lu.corbo@gmail.com>\nPaul Hovey <paul@paulhovey.org>\nCharles Corbett <nafredy@gmail.com>\nTilo Prütz <tilo@pruetz.net>\nStephen Houston <smhouston88@gmail.com>\nStorm Hess <stormhess@gloryskulls.com>\nStuart Scott <stuart.murray.scott@gmail.com>\nJacob Alzén <jacalz@tutanota.com>\nCharles A. Daniels <charles@cdaniels.net>\nPablo Fuentes <f.pablo1@hotmail.com>\nChangkun Ou <hi@changkun.de>\n\n"),
```

Comment:
We never use `StaticName` here. Would it not be nicer to just use `go:embed` and store the file data that way (more automated)?

Reply:
We cannot as they will not allow "../../" in the file path.

Reply:
Alright 👍